src/js/popup/popup.js

```diff
@@ -699,6 +764,234 @@
+             // store position as data attribute
+            input.setAttribute('data-pos',i);
+             
+            // add click handler to change position
+            input.addEventListener('click', async (e)=>{ 
+              await popup.panel.phoneMasks.utils.updateForwardingState(relayNumberData.id, e.target.getAttribute('data-forwarding') === "true");
+              await popup.panel.phoneMasks.utils.getRelayNumberData();
+              popup.panel.phoneMasks.utils.setForwardingState();
+              await popup.panel.phoneMasks.utils.updateRelayNumberStateDescription();
+              segmentedControlGroup.style.setProperty('--options-active',e.target.getAttribute('data-pos'));
+            });
+ 
+            i++;
+          });
+
+          // add class to enable the sliding pill animation, otherwise it uses a fallback
+          segmentedControlGroup.classList.add('useSlidingAnimation');
+
+          const copyRelayNumberButton = document.getElementById("fx-relay-phone-mask-copy-button");
+          const copyRelayNumberSuccessMessage = document.getElementById("fx-relay-number-copy-success");
+
+          copyRelayNumberButton.addEventListener("click", (e)=> {
+            e.preventDefault(); 
+
+            // Copy relay number to clipboard, remove first 2 characters (country code). 
+            navigator.clipboard.writeText(relayNumberData.number.substring(2, relayNumberData.number.length));
+
+            // Show success message
+            copyRelayNumberSuccessMessage.classList.add("is-shown");
+            setTimeout(() => {
+              copyRelayNumberSuccessMessage.classList.remove("is-shown")
+            }, 1000);
+          }, false); 
+ 
+        },
+        utils: {
+          updateRelayNumberStateDescription: async () => {
+            const getRelayNumber = await browser.storage.local.get("relayNumbers"); 
+            const data = popup.utilities.isNumberDataValid(getRelayNumber.relayNumbers);
+            const forwardingStateDescription = document.querySelector(".fx-relay-phone-meta-description");  
+            forwardingStateDescription.textContent = data.enabled ? browser.i18n.getMessage("popupPhoneMasksMetaForwardingDescription") : browser.i18n.getMessage("popupPhoneMasksMetaBlockingDescription");
+          },
+          setRelayNumberCountryDetails: (data) => {
+            const locationContainer = document.getElementById("fx-relay-user-phone-country");
+            const countryImage = locationContainer.querySelector("img");
+            const countryDetails = locationContainer.querySelector("span");
+            const countryCode = data.number.substring(0, 2);
+            const formattedCountryLabel = {
+              "US": "U.S.A",
+            };
+            const countryLabel = formattedCountryLabel[data.country_code] ? formattedCountryLabel[data.country_code] : data.country_code;
+            countryImage.src = `/icons/${data.country_code.toLowerCase()}-flag.svg`;
```

Comment:
Causes a console error for Canadians 😢 🇨🇦  (additional comment: Can we make the width larger?)
![image](https://github.com/mozilla/fx-private-relay-add-on/assets/59676643/4ebc95b7-adc8-429c-a6cb-f66bed2cd466)

Reply:
Once we get the icon from design team, we can get this fixed.

Reply:
Addressed - also reduced horizontal padding.